examples/userapi/04_boundary_conditions.ipynb

```diff
@@ -41,7 +41,7 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    so = 6  # Space order
</pre><p>Do you really need 6 and not something usual like 4 or 8?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/2245/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Doesn't really matter I think